src/Native/build.cmd

```diff
@@ -37,7 +36,8 @@ if exist %_VSWHERE% (
 if not exist "%_VSCOMNTOOLS%" set _VSCOMNTOOLS=%VS140COMNTOOLS%
 if not exist "%_VSCOMNTOOLS%" goto :MissingVersion
 
-set VSCMD_START_DIR="%__currentScriptDir%"
+
+set VSCMD_START_DIR=%__currentScriptDir%
```

Comment:
Why is this set different than the 2 below?  For example, the two below have the format `set "name=%__currentScriptDir%"` where this one doesn't have the surrounding double-quotes.

Reply:
You're right. I've added the quotes again (also around the variable for consistency).